lib/logging/log.js

```diff
@@ -0,0 +1,549 @@
+ *   });
+ *
+ * //-
+ * // If you anticipate many results, you can end a stream early to prevent
+ * // unnecessary processing and API requests.
+ * //-
+ * log.getEntries()
+ *   .on('data', function(service) {
+ *     this.end();
+ *   });
+ */
+Log.prototype.getEntries = function(options, callback) {
+  if (is.function(options)) {
+    callback = options;
+    options = {};
+  }
+
+  options = extend({
+    filter: format('logName="{formattedName_}"', this)
+  }, options);
+
+  return this.parent.getEntries(options, callback);
+};
+
+/**
+ * Write a log entry with a severity of "INFO".
+ *
+ * This is a simple wrapper around {module:logging/log#write}. All arguments are
+ * the same as documented there.
+ *
+ * @example
+ * var entry = log.entry('compute.googleapis.com', {
+ *   user: process.env.USER
+ * });
+ *
+ * log.info(entry, function(err, apiResponse) {});
+ */
+Log.prototype.info = function(entry, options, callback) {
+  this.write(Log.assignSeverityToEntries_(entry, 'INFO'), options, callback);
+};
+
+/**
+ * Write a log entry with a severity of "NOTICE".
+ *
+ * This is a simple wrapper around {module:logging/log#write}. All arguments are
+ * the same as documented there.
+ *
+ * @example
+ * var entry = log.entry('compute.googleapis.com', {
+ *   user: process.env.USER
+ * });
+ *
+ * log.notice(entry, function(err, apiResponse) {});
+ */
+Log.prototype.notice = function(entry, options, callback) {
+  this.write(Log.assignSeverityToEntries_(entry, 'NOTICE'), options, callback);
+};
+
+/**
+ * Write a log entry with a severity of "WARNING".
+ *
+ * This is a simple wrapper around {module:logging/log#write}. All arguments are
+ * the same as documented there.
+ *
+ * @example
+ * var entry = log.entry('compute.googleapis.com', {
+ *   user: process.env.USER
+ * });
+ *
+ * log.warning(entry, function(err, apiResponse) {});
+ */
+Log.prototype.warning = function(entry, options, callback) {
+  this.write(Log.assignSeverityToEntries_(entry, 'WARNING'), options, callback);
+};
+
+/**
+ * Write log entries to Cloud Logging.
+ *
+ * While you may write a single entry at a time, batching multiple entries
+ * together is preferred to avoid reaching the queries per second limit.
+ *
+ * @resource [entries.write API Documentation]{@link https://cloud.google.com/logging/docs/api/ref_v2beta1/rest/v2beta1/entries/write}
+ *
+ * @param {module:logging/entry|module:logging/entry[]} entry - A log entry, or
+ *     array of entries, to write.
+ * @param {object=} options - Configuration object.
+ * @param {object[]} options.labels - Labels to set on the log.
+ * @param {object} options.resource - A default monitored resource for entries
+ *     where one isn't specified.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * var entry = log.entry('compute.googleapis.com', {
```

Comment:
This code section still uses "services" (here in commends and in the method below).